source/openpulse/tests/conftest.py

```diff
@@ -0,0 +1,35 @@
+# Copied from openqasm3. OpenPulse parser should be able to parse openqasm3 examples
+import collections
+import pathlib
+
+import pytest
+
+import openpulse
+
+TEST_DIR = pathlib.Path(__file__).parent
+ROOT_DIR = TEST_DIR.parents[2]
+EXAMPLES_DIR = ROOT_DIR / "examples"
+# The following 2 examples need to fixed to match the latest openpulse specification
+EXCLUDES = ["defcal.qasm", "pong.qasm"]
```

Comment:
Skipping these 2 examples because the defcal body cannot be passed by openpulse, e.g, `gaussian(...)`.

Reply:
If those examples are valid OpenPulse, we probably need to update those examples.

Reply:
Have these been updated?

Reply:
@jakelishman removed `pong.qasm` in #376 . However, `defcal.qasm` still contains some abbreviations such as:

```
defcal x $0 {
   play drive($0), gaussian(...);
}
```

This example was from a [couple years ago](https://github.com/openqasm/openqasm/commit/8c5d723aee624a8f044dbceb469d3ae483d36029#diff-e54db80c3f513f2802d83657bc1316420cc14a08bd3c943f5aace58d201e5e50) and the syntax has changed significantly so we should consider to create a separate issue to fix this example.